api/swagger.json

```diff
@@ -0,0 +1,1304 @@
+            "description": "Location id to delete",
+            "required": true,
+            "schema": {
+              "type": "integer",
+              "format": "int64"
+            }
+          }
+        ],
+        "responses": {
+          "400": {
+            "description": "Invalid ID supplied"
+          },
+          "401": {
+            "description": "Not authorized"
+          },
+          "404": {
+            "description": "Location not found"
+          },
+          "default": {
+            "description": "unexpected error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/chapters": {
+      "post": {
+        "tags": [
+          "chapter"
+        ],
+        "summary": "Add a new chapter",
+        "description": "",
+        "operationId": "addChapter",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Chapter"
+              }
+            }
+          },
+          "description": "Location coordinates",
+          "required": true
+        },
+        "responses": {
+          "200": {
+            "description": "Chapter response",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Chapter"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Not authorized"
+          },
+          "default": {
+            "description": "unexpected error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/chapters/{chapterId}": {
+      "get": {
+        "tags": [
+          "chapter"
+        ],
+        "summary": "Retreive the details of a chapter",
+        "description": "",
+        "operationId": "findChapterById",
+        "parameters": [
+          {
+            "name": "chapterId",
+            "in": "path",
+            "description": "ID of chapter to return",
+            "required": true,
+            "schema": {
+              "type": "integer",
+              "format": "int64"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Chapter"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Not authorized"
+          },
+          "400": {
+            "description": "Invalid ID supplied"
```

Comment:
Same 400 vs. 404 as mentioned for locations. Also applies to PATCH/DELETE and subresources.